changelog/114.feature.rst

```diff
@@ -0,0 +1,3 @@
+Addition of the `sunkit_image.granule` module and contains functions to segment granulation in images of the solar photosphere.

```

Comment:
```suggestion
Added the `sunkit_image.granule` which provides functions to segment granulation in images of the solar photosphere.

```

Reply:
I apologize if I am correcting my own corrections here (I know I previously asked for a rewording which may be why some of the language got a bit jumbled).